assets/app/features/upgrade/i18n/en.json

```diff
@@ -76,6 +76,7 @@
             },
             "database-configuration": {
                 "title": "Enter Credentials and/or Information to create or connect to a PostgreSQL service to be used for serving the Crowbar database",
+                "title-completed": "This step has been completed. Click on button labeled \"Next\" to continue the upgrade process.",
```

Comment:
For base version I directly took what was proposed. I didn't really like this wording either. I changed it now to better match the rest of places (which are not really 100% consistent regarding clicking (on) buttons).